smrf/framework/model_framework.py

```diff
@@ -217,6 +202,11 @@ def _setup_date_and_time(self):
         ))
         self.time_steps = len(self.date_time)
 
+    @property
+    def thermal_netcdf(self):
+        return self.distribute['thermal'].gridded and \
+            self.config['gridded']['data_type'] in ['wrf', 'netcdf']
```

Comment:
`['wrf', 'netcdf']` smells like a good candidate for a class constant.

Reply:
This method was for the original thermal implementation where we're using the thermal from the atmospheric model. We've switched that with HRRR to calculated clear sky thermal and I'd like to see all the other gridded datasets do that as well to keep it consistent.